client/src/components/App.jsx

```diff
@@ -90,22 +90,24 @@ class App extends React.Component {
 
   render() {
     return (
-      <ThemeProvider theme={theme}>
-        <>
-          <GlobalStyles/>
-          <Container>
-            <h1>Welcome to Atelier!</h1>
-            <Overview
-              currentProductId={this.state.currentProductId}
-              currentProductRating = {this.state.currentProductRating} />
-            {/* <div><Reviews currentProductId = '40344'currentProductRating = {this.state.currentProductRating}/></div> */}
-            <br/>
-            <div><QA productId={this.state.currentProductId}
-              sessionCookie={this.state.sessionCookie} addToCookie={this.addToCookie}/>
-            </div>
-          </Container>
-        </>
-      </ThemeProvider>
+      <div>
```

Comment:
What is the use of another div outside of ThemeProvider ?

Reply:
I used it in the App.test.js file to make sure it rendered a div type, since I'm not sure if you can test if it rendered a type that is a styled component